src/stores.ts

```diff
@@ -64,3 +71,25 @@ export function deleteIntervention(id: number) {
   sidebarHover.set(null);
   openFromSidebar.set(null);
 }
+
+function loadUserSettings(): UserSettings {
+  let settings = {
+    streetViewImagery: "google",
+    avoidDoublingBack: false,
+  };
+
+  // Be paranoid when loading from local storage, and only copy over valid items
```

Comment:
Maybe this is overkill. As we evolve ATIP, these settings will change over time, but compatibility won't be that hard to handle. Just use new fields if things change in some crazy way. Think we should simplify it?